schemas/x509.yml

```diff
@@ -52,8 +52,8 @@
       type: keyword
       short: Unique serial number issued by the certificate authority.
       description: >
-        Unique serial number issued by the certificate authority. For consistency, if this value is alphanumeric, it should be
-        formatted without colons and uppercase characters.
+        Unique serial number issued by the certificate authority. For consistency, this should be
```

Comment:
Given this is `should`, it implies this encoding is still optional, which might not really completely resolve the original problem. Is that what you intend?  But if the wording is changed to something stronger, it could be considered a breaking change.

If you want it to be enforced, what do you think of having `should` in ECS v8, and changing to `must` in the next major release?

Reply:
I agree, that making it a MUST would be breaking for v8 so I think it needs to remain a SHOULD.

About changing it in the next major, sounds like a good idea.

Reply:
I'm fine with that, I'm not familiar with the process at all.
For the next release, shouldn't we restrict it even more? I'd prefer to have something that is strict on case, semi-colons and base.